### ISSUE TITLE
Twelve of Code information page for before 1.0.0

### ISSUE THREAD

**Issue by BeepBot99** [Mesure73L/mesure.x10.mx#113]
### What problem were you having?

If someone goes to <https://mesure.x10.mx/twelve-of-code>, they will see an unfinished, half-completed version of Twelve of Code.

### What solution would you like?

Make a new page (I can do this if you want) for Twelve of Code that is just temporary. Put it in the `main` branch, and then when `develop` is merged into `main`, it will go away.
The new page will talk about what Twelve of Code is, and it will say when it will be released.

### What alternatives have you considered?

Just keep the unfinished version of Twelve of Code until March 1, 2023

#### Pros:

- No one will see and unfinished, uncompleted version of Twelve of Code ever.
- It can help with outreach before Twelve of Code is released.
- It could attract new developers to help with Twelve of Code 1.0.0.

#### Cons:

- It would take time to make, and the full Twelve of Code is more important right now.
- There would be a merge conflict when merging `develop` into `main`.
- It would be another place for Minystreem to complain about how bad Twelve of Code is.

### What page is this for?

Twelve of Code

**Comment by Mesure73L** [comment]
Good idea, not sure if it's gonna happen tho, like you mentioned our highest priority right now is working on 1.0.0

**Comment by BeepBot99** [comment]
When we created the `develop` branch from `main`, Twelve of Code in `main` was an older version of Twelve of Code. If we change it, then the version of Twelve of Code in `main` will not be the same as the version of Twelve of Code in `main` when we created `develop` from it. Therefore, there will be a merge conflict. Although, it will be fine, because we will just completely overwrite it with the content from `develop`.

**Comment by Mesure73L** [comment]
If we create this, I will do it

**Comment by BeepBot99** [comment]
No, I can do this. Or, we can both do it, but mine will be better.
<table>
<thead>
<tr>
<th>Person whose is better</th>
<th>Chance of that happening</th>
</tr>
</thead>
<tbody>
<tr>
<td>BeepBot99</td>
<td>100%</td>
</tr>
<tr>
<td>Mesure73L</td>
<td>0%</td>
</tr>
</tbody>
</table>

**Comment by Mesure73L** [comment]
> No, I can do this. Or, we can both do it, but mine will be better.
> 
> Person whose is better	Chance of that happening
> BeepBot99	100%
> Mesure73L	0%

Hey.
I want to do this.

**Comment by Mesure73L** [comment]
| Person | Change of being kicked |
|--------|--------|
| BeepBot99 | 100% |
| Mesure73L | 0% |

**Comment by BeepBot99** [comment]
@Mesure73L What about something like this?
https://twelve-of-code-about.netlify.app/

**Comment by BeepBot99** [comment]
And then I could put a timer under it and an about section under that.

**Comment by Mesure73L** [comment]
***I*** will make this.

**Comment by BeepBot99** [comment]
Cool.
I will too.

**Comment by BeepBot99** [comment]
@Mesure73L Can I please do this? I really want to.

**Comment by Mesure73L** [comment]
You can make one, it just won't be the official one.

**Comment by BeepBot99** [comment]
But what if it's better? Also, I can promise to have a finished one by next Tuesday.

**Comment by Mesure73L** [comment]
Then I don't care.

**Comment by BeepBot99** [comment]
Why do you care, you're not using it.
If you're going to make this website, I do not want you to copy code from mine.

**Comment by Mesure73L** [comment]
@BeepBot99 you're going to show me when it's done, right?

**Comment by BeepBot99** [comment]
Only if you've already finished. I am not showing you before you have finished.

**Comment by Mesure73L** [comment]
k

**Comment by BeepBot99** [comment]
Can you make a page for now, and I make a page similar to the one I have already made for after 1.0.0 releases?

**Comment by Mesure73L** [comment]
I kinda want to see what your ideas were after I finish mine tho

**Comment by BeepBot99** [comment]
Well, if I can do one for only after 1.0.0, I might not finish the one I am currently making with the timer, and instead do a more permanent one.
